Cargo.toml

```diff
@@ -131,6 +122,18 @@ object_store = { version = "0.9.0", default-features = false }
 [workspace.dependencies]
 async-trait = { version = "0.1.74" }
 axum = { version = "0.7.1", features = ["macros"] }
+tower = "0.4"
+tower-http = { version = "0.5.0", features = [
+  "trace",
+  "catch-panic",
+  "timeout",
+  "add-extension",
+  "cors",
+  "fs",
+  "set-header",
+  "compression-full",
+  "normalize-path"
```

Comment:
This is not a requirement from loco framework, this means that the `normalize-path` will includded for all the users even if they not using your initializer. can you please move it to the loco-extras cargo file?

Reply:
Done. Moved to the `loco-extras` cargo file.